docs/component-guides/kserve.md

```diff
@@ -0,0 +1,129 @@
+
+As you understand from the [background](#background) section, we need to use a custom domain to configure HTTPS with load balancer. We recommend using HTTPS to to enable traffic encryption between the clients and your load balancer. So, if you are using `platform.example.com` domain to host Kubeflow, you will need to edit the `config-domain` ConfigMap in the `knative-serving` namespace to configure the `platform.example.com` to be used as the domain for the routes. **Follow the instructions** in the [procedure section of the using a custom domain guide](https://knative.dev/docs/serving/using-a-custom-domain/#procedure) on the Knative Serving documentation to configure the default domain used by Knative. You would remove the `_example` key and replace `example.com` with your domain e.g. `platform.example.com`. Following is a sample:
+
+```
+apiVersion: v1
+kind: ConfigMap
+data:
+  platform.example.com: ""
+...
+```
+
+### Create a certificate to use with Load Balancer
+
+To get TLS support from the ALB, you need to request a certificate in AWS Certificate Manager. Since Knative concatenates the namespace in the FQDN for a route and it is delimited by a dot by default, the URLs for `InferenceServices` created in each namespace will be in a different [subdomain](https://en.wikipedia.org/wiki/Subdomain). For example, if you have 2 namespaces, `staging` and `prod` and create an `InferenceService` resource called `sklearn-iris` in both the namespaces, the URLs for each resource will be `http://sklearn-iris.staging.platform.example.com` and `http://sklearn-iris.prod.platform.example.com` respectively. This means, you will need to specify all the subdomains in which you plan to create InferenceService while creating the SSL certificate in ACM. For example, in the case with staging and prod namespaces, you will need to add `*.prod.platform.example.com`, `*.staging.platform.example.com` and `*.platform.example.com` to the certificate. This is because DNS only supports wildcard in the [leftmost part of the domain name](https://en.wikipedia.org/wiki/Wildcard_DNS_record) e.g. `*.platform.example.com`. When you request a wild-card certificate, the asterisk (*) must be in the leftmost position of the domain name and can protect only one subdomain level. For example, `*.platform.example.com` can protect `staging.platform.example.com`, and `prod.platform.example.com`, but it cannot protect `sklearn-iris.staging.platform.example.com`.
+
+For this tutorial, create an ACM certificate for `*.platform.example.com` and `*.staging.platform.example.com`(**both domains in the same certificate**) in the region where your cluster exists by following the process similar to [create certificates for domain](../deployment/add-ons/load-balancer/README.md#create-certificates-for-domain) section of the load balancer guide. Once the certificate status changes to `Issued`, export the ARN of the certificate created:
```

Comment:
could we include a describe output of the certificate or an image as an example to show what the expected output could look like?